### PR TITLE
Finrem-cos Flux V2 prod

### PR DESCRIPTION
**Pull request by damongreen123**
https://tools.hmcts.net/jira/browse/DTSSE-1264

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by timja — CHANGES_REQUESTED** [review]
you haven't added it to all bases, this just removes the app

**Comment by damongreen123** [comment]
> 

yep, still working on this one Tim

**Review by ruban72 — DISMISSED** [review]

**Review by hannah38 — APPROVED** [review]

File: apps/financial-remedy/finrem-cos/prod.yaml [hunk 1]
```diff
@@ -11,4 +12,6 @@ spec:
         ASSIGNED_TO_JUDGE_DEFAULT_EMAIL: newApplication@justice.gov.uk
         FEATURE_PBA_CASE_TYPE: false
         FEATURE_USE_USER_TOKEN: true
-        VAR_T2: trigger2
\ No newline at end of file
+        VAR_FLUX_V2: trigger3
+      autoscaling:
+        minReplicas: 3
```
Comment (timja) [inline apps/financial-remedy/finrem-cos/prod.yaml:17]: minReplicas is not something we offer.

it uses `replicas` and `maxReplicas`

https://github.com/hmcts/chart-library/blob/master/library/templates/v2/_hpa.tpl#L11-L12

**Review by timja — CHANGES_REQUESTED** [review]

File: apps/financial-remedy/finrem-cos/prod.yaml (hunk 1 above)
Comment (damongreen123) [reply apps/financial-remedy/finrem-cos/prod.yaml]: removed that

**Review by timja — APPROVED** [review]

**Review by ruban72 — APPROVED** [review]

**Pull request merged**